src/curated-corpus/components/ScheduleHistory/ScheduleHistory.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Card, Grid, Typography } from '@material-ui/core';
+import { DateTime } from 'luxon';
+import { ApprovedCorpusItemScheduledSurfaceHistory } from '../../../api/generatedTypes';
+import { useStyles } from './ScheduleHistory.styles';
+import { getCuratorNameFromLdap } from '../../helpers/helperFunctions';
+import { ScheduledSurfaces } from '../../helpers/definitions';
+
+interface ScheduleHistory {
+  data: ApprovedCorpusItemScheduledSurfaceHistory[];
+}
+
+export const ScheduleHistory: React.FC<ScheduleHistory> = (
+  props
+): JSX.Element => {
+  const classes = useStyles();
+  const { data } = props;
+
+  const getScheduledSurfaceName = (surfaceGuid: string) => {
+    return (
+      ScheduledSurfaces.find((surface) => surface.guid === surfaceGuid)?.name ??
+      'Unknown Surface'
+    );
+  };
```

Comment:
Would this ever happen in practice? Should we throw an error here for Sentry to highlight a data integrity issue? 

Would it be better to show the GUID provided instead of "Unknown Surface" (I am thinking if we have one extra surface added on the backend but the frontend is not updated, it would at least show the code/GUID for that surface instead of "Unknown Surface".